lib/vmpooler/metrics/promstats.rb

```diff
@@ -159,82 +146,72 @@ def vmpooler_metrics_table
             mtype: M_COUNTER,
             torun: %i[manager],
             docstring: 'vmpooler machine migrated from',
-            prom_metric_prefix: "#{@metrics_prefix}_migrate_from",
             param_labels: %i[host_name]
           },
           migrate_to: {
             mtype: M_COUNTER,
             torun: %i[manager],
             docstring: 'vmpooler machine migrated to',
-            prom_metric_prefix: "#{@metrics_prefix}_migrate_to",
             param_labels: %i[host_name]
           },
           api_vm: {
             mtype: M_COUNTER,
             torun: %i[api],
             docstring: 'Total number of HTTP request/sub-operations handled by the Rack application under the /vm endpoint',
-            prom_metric_prefix: "#{@metrics_prefix}_http_requests_vm_total",
+            prom_metric_id: 'http_requests_vm_total',
```

Comment:
Why is this one different from all the others?

Reply:
Thats actually a good question - should I should change it in the ```.increment``` statements``` ?
Probably 😀

Reply:
Fixed the one anomaly and simplified this.